management/management_request.go

```diff
@@ -132,6 +132,10 @@ type List struct {
 
 // HasNext returns true if the list has more results.
 func (l List) HasNext() bool {
+	if l.Next != "" {
+		return true
+	}
+
 	return l.Total > l.Start+l.Limit
```

Comment:
Even though we could rewrite this to simply `return l.Next != "" || l.Total > l.Start+l.Limit` I prefer how you've done it so it's easier to understand what's happening and is less cluttered 👍🏻